CruCentralCoast/Community/Missions/MissionsTableVC.swift

```diff
@@ -10,18 +10,28 @@ import UIKit
 
 class MissionsTableVC: UITableViewController {
     
-    //test data array
-    var testDataArray: [MissionCellParameters] = [MissionCellParameters(titleLabel: "Oasis", date: "11/11/11", location: "test location", description: "test description"),MissionCellParameters(titleLabel: "Oasis", date: "11/11/11", location: "test location", description: "test description"),MissionCellParameters(titleLabel: "Oasis", date: "11/11/11", location: "test location", description: "test description"),MissionCellParameters(titleLabel: "Oasis", date: "11/11/11", location: "test location", description: "test description"),MissionCellParameters(titleLabel: "Oasis", date: "11/11/11", location: "test location", description: "test description"),MissionCellParameters(titleLabel: "Oasis", date: "11/11/11", location: "test location", description: "test description")]
+    var dataArray = [Missions]()
 
     override func viewDidLoad() {
         super.viewDidLoad()
         
         self.navigationController?.navigationBar.prefersLargeTitles = true
 
-        // Configure the cell...
-        let nib = UINib.init(nibName: "CommunityTableViewCell", bundle: nil)
-        self.tableView.register(nib, forCellReuseIdentifier: "comCell")
-
+        self.tableView.registerCell(CommunityTableViewCell.self)
+        
+        DatabaseManager.instance.getMission{ (mission) in
```

Comment:
"mission" should be "missions" since it is an array of Mission objects. The for loop below this then makes more sense as "for mission in missions"